### ISSUE TITLE
Remove 'uses' and 'conversion' annotations from `Modelica.Electrical.Spice3`

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#312]
**Reported by dietmarw on 11 Mar 2010 21:23 UTC**
Should be cleaned up asap:

```
./Modelica/Electrical/Spice3.mo:240:      uses(Modelica(version="3.1")));
./Modelica/Electrical/Spice3.mo:1455:        uses(                     Modelica(version="3.1"), Spice3(version="1")),
./Modelica/Electrical/Spice3.mo:8453:    uses(Modelica(version="3.1")),
./Modelica/Electrical/Spice3.mo:8454:    version="1");
```

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/312

**Comment by modelica-trac-importer** [comment]
**Comment by dietmarw on 12 Mar 2010 11:39 UTC**
Fixed in ac1f8b3b5573430581e058e326c8bddcf883900c.

**Comment by modelica-trac-importer** [comment]
**Comment by dietmarw on 12 Mar 2010 12:13 UTC**
One additional "uses" annotation was overlooked. Fixed in e01f11ed6dccf20de024e384a91bcb6e24736be7.
